config/log4j2.properties

```diff
@@ -0,0 +1,36 @@
+status = error
+name = LogstashPropertiesConfig
+
+property.filename = /tmp/logstash/logstash.log
```

Comment:
I am not completely familiar with the **log4j** properties files,  but we already have a similar option in the settings.yml and we use them for the logger. Is that option necessary? It makes the assumption that this path exist (windows don't have this.). Same issue with the `rolling.filePattern` probably.

https://github.com/elastic/logstash/pull/5651/files#diff-da023432533015a65b7931c1c17d4c60R168